docs/current/cli/698277-index.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: /cli
+---
+
+# Dagger CLI
+
+<div class="status-badge">Technical Preview</div>
+
+## What is the Dagger CLI?
+
+The Dagger CLI is an optional tool to interact with the Dagger Engine in different ways. This includes:
+
+* Developing and running CI/CD pipelines from the command-line.
+* Proxying requests to the Dagger GraphQL API.
+* Executing custom commands in a Dagger session for deeper introspection and debugging.
+
+Currently the CLI consists of:
+
+* A binary file
+* This documentation
+
+In practice, the CLI is optional and only necessary for specific use cases, as described above.
+
+## Who is it for?
+
+The Dagger CLI may be a good fit if you are...
+
+* A "designated devops person" who doesn't think of themselves as a developer - nothing makes you happier than a battle-hardened shell script.
+* A GraphQL veteran looking to connect existing GraphQL tools and libraries with Dagger's API.
+
+The Dagger CLI may *not* be a good fit if you are...
+
+* A developer who doesn't know shell scripting, and is not interested in learning it.
+* Someone who loves writing YAML all day, thank you very much.
+* A container skeptic: the less containers are involved, the happier you are.
```

Comment:
I think we should cut this part. The CLI is different from the other sections, as every Dagger user may benefit from using, although not always for the same reasons. In that way it is different from both the SDKs and GraphQL API.

For example:

* Non-developers can use it to interact with the Dagger engine from the terminal or a shell script
* Developers using an SDK can use it as a complement to their development experience (trying out queries, debugging their code, etc)
* Developers *not* using an SDK can use it in the same way, but will need it even more given their reliance on the GraphQL API.

Reply:
Deleted